discussion/app/discussion/util/Http.scala

```diff
@@ -1,20 +1,20 @@
 package discussion.util
 
 import play.api.libs.ws.{WS, WSResponse}
-import play.api.libs.json.{Json, JsValue}
-import common.Logging
-import scala.concurrent._
-import ExecutionContext.Implicits.global
+import play.api.libs.json.{JsValue, Json}
+import common.{ExecutionContexts, Logging, StopWatch}
+import scala.concurrent.Future
 
-trait Http extends Logging {
+trait Http extends Logging with ExecutionContexts {
```

Comment:
Taking advantage of this PR to use the Play default ExecutionContext rather than the scala one